sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -1126,6 +1139,7 @@ public partial class KeywordTokenizer : Azure.Search.Documents.Indexes.Models.Le
     public partial class KnowledgeStore
```

Comment:
>KnowledgeStore [](http://example.com/codeflow?start=25&length=14)

Is this a new auth mode?

Reply:
No, KnowledgeStore is used to store AI-enriched content in tables and blob containers in Azure Storage.

[Here's](https://learn.microsoft.com/en-us/azure/search/knowledge-store-concept-intro?tabs=portal) more about Knowledge store in Azure Cognitive Search.